docs/pages/enterprise/getting-started.mdx

```diff
@@ -214,12 +214,33 @@ role only allows SSH logins as `root@host`.
 ```yaml
 kind: role
 metadata:
-  name: access
+  name: access-sample
 spec:
   allow:
-    logins:
-    - '{{internal.logins}}'
-    - root
+    # The logins array defines the OS/UNIX logins a user is allowed to use
```

Comment:
I think this can be clearer about how `internal.logins` is populated. "DB user" isn't a widely used term in the docs, so I'm not sure readers will understand it. Maybe:

```yaml
    # The logins array defines the OS/UNIX logins a user is allowed to use
    # for accessing Nodes in the Teleport cluster. If a Teleport user with this
    # role is a local user, i.e., was created via `tctl`, the Auth Service populates
    # the `{{internal.logins}}` template variable with the user's
    # `logins` trait. For example, if the user has the logins "ec2-user,ubuntu",
    # the Auth Server replaces `{{internal.logins}}` with "ec2-user,ubuntu".
    # This means that you can change the logins you assign to your Teleport 
    # users without needing to update their roles.
    #
    # If a remote user is accessing a Trusted Cluster, the cluster's Auth Service
    # replaces `{{internal.logins}}` with the remote user's logins.
```